build/tools/Automation/Extensions/HttpClientExtensions.cs

```diff
@@ -0,0 +1,83 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace Microsoft.Oryx.Automation.Extensions
+{
+    public static class HttpClientExtensions
+    {
+        /// <summary>
+        /// Sends a GET request to the specified URL using the given HttpClient
+        /// instance and returns the response content as a string.
+        /// Returns null if the response status code is not successful.
+        /// </summary>
+        /// <param name="httpClient">The HttpClient instance to use for the request.</param>
+        /// <param name="url">The URL to send the request to.</param>
+        /// <returns>A Task<string> representing the asynchronous operation. The result of
+        /// the task is the response content as a string, or null if the response status code is not
+        /// successful.
+        /// </returns>
+        public static async Task<string> GetDataAsync(this HttpClient httpClient, string url)
+        {
+            try
+            {
+                Console.WriteLine("Making GET request to: " + url);
+                HttpResponseMessage response = await httpClient.GetAsync(url);
+                Console.WriteLine($"Response received.: {response.StatusCode}");
```

Comment:
nit: would you mind removing the `.` character before the `:` character on this line?

```
Response received: {response.StatusCode}
```

Reply:
Remove